content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -1,146 +1,404 @@
-orders and in your eCommerce catalog. To do so:
+At the bottom of the :guilabel:`Create Pricelist Rules` pop-up form is the :guilabel:`Conditions`
+section.
 
-* Check *Allow discounts on sales order lines* in :menuselection:`Sales --> Configuration -->
-  Settings --> Quotations & Sales --> Discounts`.
-* Apply the option in the pricelist setup form.
+Here, start by selecting one of the options in the :guilabel:`Apply On` field:
 
-.. image:: pricing/discount_options.png
+- :guilabel:`All Products`: the advanced pricelist rule will be applied to all products.
+- :guilabel:`Product Category`: the advanced pricelist rule will be applied to a specific category
+  of products.
+- :guilabel:`Product`: the advanced pricelist rule will be applied to a specific product.
+- :guilabel:`Product Variant`: the advanced pricelist rule will be applied to a specific product
+  variant.
+
+If any of those options, apart from :guilabel:`All Products`, is selected, a new option-specific
+field appears, in which the specific :guilabel:`Product Category`, :guilabel:`Product`, or
+:guilabel:`Product Variant` must be chosen, depending on which :guilabel:`Apply On` option is
+chosen.
+
+Then, select a minimum quantity to be applied to the advanced pricelist rule in the :guilabel:`Min.
+Quantity` field. And, lastly, select a range of dates for the pricelist item validation in the
+:guilabel:`Validity` field.
+
+Once all configurations are complete, either click :guilabel:`Save & Close` to save the advanced
+pricelist rule, and return to the :guilabel:`Pricelists` page. Or, click :guilabel:`Save & New` to
+immediately create another advanced pricelist rule on a fresh, completely blank :guilabel:`Create
+Pricelist Rules` pop-up form.
+
+.. note::
+   If a rule is set for a particular item, and another one for its category, Odoo takes the rule of
+   the item. Make sure at least one pricelist item covers all your products.
+
+Show discount percentage to customers
+=====================================
+
+With Odoo *Sales* the option to show the public price *and* the computed discount percentage on
+printed sales orders and in the *eCommerce* product catalog is available.
+
+To do that, navigate to :menuselection:`Sales app --> Configuration --> Settings`, and in the
+:guilabel:`Pricing` section, click the checkbox next to the :guilabel:`Discounts` feature, and then
+click :guilabel:`Save` to save all changes.
+
+After activating the :guilabel:`Discounts` feature, navigate to the :guilabel:`Pricelists` page,
+either by clicking the :guilabel:`Pricelists` page on the *Sales* :guilabel:`Settings` page, or
+navigate to :menuselection:`Sales app --> Products --> Pricelists`.
+
+On the :guilabel:`Pricelists` page, select the desired pricelist to edit. On the pricelist detail
+form, click into the :guilabel:`Configuration` tab. In the :guilabel:`Configuration` tab, at the
+bottom, a new :guilabel:`Discounts` section is now available.
```

Comment:
This section is available even if the Pricelist setting is "Multiple prices per product" and even if no discount has been configured yet. So this part should be reworded slightly.